lifecycle-operator/controllers/common/phase/phasehandler.go

```diff
@@ -17,7 +18,7 @@ import (
 
 type PhaseHandler struct {
```

Comment:
as this is now in its own package, we may consider dropping the package name as a prefix for the struct, i.e. `PhaseHandler` -> `Handler` (as it will be referred to as `phase.Handler` in other packages. This would be more in line with the naming conventions in go. The same goes for other packages, such as the `EvaluationHandler` in the `evaluation` package. I'll leave it up to you though if you would like to do that in this PR or if we should do a follow up, if we would like to change the naming

Reply:
good point, will adapt :+1: